test/backend/test.py

```diff
@@ -877,7 +965,12 @@ def run(self, inputs, **kwargs):
             if (sys_is_le != inp_is_le):
                 inputs = list(
                     map(lambda x: x.byteswap().newbyteorder(), inputs))
-            outputs = super().run(inputs)
+            # If constant test, change the model inputs to constants.
+            if args.constant:
+                inputs = self.turn_model_input_to_constant(inputs)
+                self.exec_name = self.compile_model()
```

Comment:
Can we have `onnx.save(self.model, model_name)` after the model is modified? The saved model may be used for debugging.

Reply:
We did have it. `onnx.save` is inside `compile_model`.

Reply:
From the name of method and the order of statement, I thought the compile_model is before run_model where the model is changed for constant input.